data/human/hails.txt

```diff
@@ -1568,6 +1569,107 @@ phrase "friendly civilian"
 		"Apparently people think we'll work for exposure."
 		"Not to mention this ship is not designed for a job like it."
 
+phrase "friendly civilian"
+	word
+		"Don't go for"
+		"Don't participate in"
+		"Stay away from"
+		"Watch out for"
+	word
+		" "
+	word
+		"badly organized races"
+		"inheritance scams"
+		"shady transport jobs"
+		"hitman transports"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"I lost 20000 credits on one of those."
+		"I nearly got killed in one."
+		"The Navy almost arrested me."
+		"Those can ruin your life!"
+		"They are usually too good to be true."
+		"They take your money and you won't get anything in return."
+
+phrase "friendly civilian"
+	word
+		"I have a delivery for"
+		"I'm looking for"
+		"My passengers need to go to"
+	word
+		" "
+	word
+		"Dune"
+		"Earth"
+		"Foundry"
+		"Glaze"
+		"Moonshake"
+		"Norn"
+		"Skillet"
+	word
+		", but my starmap doesn't list it."
+		"; do you know where I can find that planet?"
+		"; have you ever heard of a planet with that name?"
```

Comment:
```suggestion
		"; have you ever heard of it?"
```
Same as above.
In regards to Anarchist's comment about Earth above and your response, I do like the idea of someone trying to brag about Earth, so I'd agree with keeping it in.